src/test/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManagerTests.java

```diff
@@ -708,6 +717,52 @@ public void update_all_project_bindings_ignore_wrong_binding_default_folder() {
     assertThat(logTester.logs(LoggerLevel.ERROR)).contains("Invalid binding for 'default folder'");
   }
 
+  @Test
+  public void update_bindings_error_client_notification() {
+    WorkspaceFolderSettings folderSettings = mock(WorkspaceFolderSettings.class);
+    WorkspaceSettings settings = mock(WorkspaceSettings.class);
+    Map<String, ServerConnectionSettings> servers = mock(Map.class);
+    String serverId = "serverId";
+    when(settingsManager.getCurrentDefaultFolderSettings()).thenReturn(folderSettings);
+    when(folderSettings.hasBinding()).thenReturn(true);
+    when(folderSettings.getConnectionId()).thenReturn(serverId);
+    when(settingsManager.getCurrentSettings()).thenReturn(settings);
+    when(settings.getServers()).thenReturn(servers);
+    ServerConnectionSettings serverConnectionSettings = new ServerConnectionSettings("serverId", "serverUrl", "token", "organizationKey");
+    when(servers.get(serverId)).thenReturn(serverConnectionSettings);
+    when(globalStorageStatus.isStale()).thenReturn(true);
+    when(enginesFactory.createConnectedEngine(serverId)).thenReturn(fakeEngine);
+    doThrow(new RuntimeException("Boom")).when(fakeEngine).update(any(ServerConfiguration.class), any());
+
+    underTest.getBinding(fileInAWorkspaceFolderPath.toUri());
+
+    verify(client, times(1)).showMessage(any());
```

Comment:
Can't you assert a bit more explicitly the message?

Reply:
Done.